src/main/java/com/kodilla/ecommercee/entity/User.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.ecommercee.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @GeneratedValue
+    private Long userId;
```

Comment:
Skoro pole jest już w encji User, to nie trzeba dodawać tego przedrostka, wystarczy samo "id" jako nazwa zamiast userId